config/settings.yml

```diff
@@ -0,0 +1,2 @@
+dfe_signin:
+  identifier: bats2
```

Comment:
not too sure should it be in repo...

Reply:
Was intentional. It's exposed through the callback URL to the client